src/components/UserSearchBar.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { Button, TextField, Typography, Link, InputBase, Paper, createStyles, Theme, withStyles } from '@material-ui/core';
+import SearchIcon from '@material-ui/icons/Search';
+interface SearchProps {
+  onSearchChange: any;
+  classes: { search: string; searchWrapper: string; inputWrapper: string; };
+}
+
+const styles = (theme: Theme) =>
```

Comment:
If the styles don't use theme, you don't need to have it as a parameter. In general, don't add a parameter unless it's going to be used (this might change since we should be using the `primary.main` from themeStyles.ts).